spec/instant.html

```diff
@@ -8,48 +8,47 @@ <h1>Instant Objects</h1>
   <emu-clause id=sec-temporal-instant-constructor>
     <h1>new Instant(*nanoseconds*)</h1>
     <p>When called as a constructor creates a new Instant object. *nanoseconds* is a BigInt of the nanoseconds since
-      epoch.</p>
+      epoch. This value is stored in the [[value]] slot.</p>
```

Comment:
I don't think Instant changes belong in this PR, but as an aside: if we want to use internal slots at all (which is debatable), [[InstantData]] or maybe [[InstantValue]] would be more consistent with the rest of ECMAScript.